inst/tinytest/testPlot.R

```diff
@@ -11,6 +11,8 @@ testStudyName <- "ABC"
 testStudyObj <- OmicNavigator:::testStudy(name = testStudyName)
 plots <- OmicNavigator:::testPlots()
 testStudyObj <- addPlots(testStudyObj, plots)
+mapping <- OmicNavigator:::testMapping()
+testStudyObj <- addMapping(testStudyObj, mapping)
```

Comment:
I think the above lines are unnecessary. I explicitly load the plots since that is the only way to get around the fact that the plotting functions need to be created in the current environment. The mapping and all the other elements are already returned by `testStudy()`

Reply:
I removed them in the next commit. Indeed, this did not influence anything on the tests. Thanks!